### ISSUE TITLE
Align code needs to be updated

### ISSUE THREAD

**Issue by Spuds** [Frenzie/elk-quick-quote#8]
See https://github.com/Spuds/elk-quick-quote/commit/bc34357a800ba2649f3fdcd01a34dd09eb1c2f68

A couple of minor things .. `left `and `right `are unique BBC codes not part of `align=` as such those props objects  need to be updated.

In seeing that, center is also a `div `class not a `span `align.  Yes this is not HTML correct, center is not a block element, but as far as the user is concerned when using the editor, center is a block element and that is how ElkArte deals with it.  Check the years of defending the way center worked vs left/right.

TLDR, we need to check for the `centertext  `class in the `div `section and not the `span `section

Updated the `checkCSSProps ` function to able to check for a class name  by setting `isClass ` value of the passed object.  This allowed the consolication of centertext and tt items. 

The above should be clear in the code and is 1.1 valid, hence the issue.

**Comment by Frenzie** [comment]
Yes, I'd noticed it needed some updates. :-)
